src/style.css

```
@@ -0,0 +1,84 @@
+html {
+  --rectangle-big-height: 70px;
+  --rectangle-small-height: 42px;
+
+  --big-image-height: 19px;
+  --small-image-height: 11px;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  margin-top: 0;
+}
+
+.form {
+  position: relative;
+  margin: 20px 0;
+}
+
+.search-image {
+  position: absolute;
+}
+
+.search-image--big {
+  height: var(--big-image-height);
+  top: calc((var(--rectangle-big-height) - var(--big-image-height)) / 2);
+  left: 17px;
+}
+
+.search-image--small {
+  height: var(--small-image-height);
+  top: calc((var(--rectangle-small-height) - var(--small-image-height)) / 2);
+  left: 13px;
+}
+
+.input {
+  font-family: "Avenir-Book", Arial, Helvetica, sans-serif;
```

move it to the html tag selector and let inputs inherit it 
```suggestion
  font-family: inherit;
```